lib/ramble/ramble/util/web.py

```diff
@@ -264,11 +264,10 @@ def url_exists(url):
 def _debug_print_delete_results(result):
     if 'Deleted' in result:
         for d in result['Deleted']:
-            tty.debug('Deleted {0}'.format(d['Key']))
+            ramble.util.logger.logger.debug(f'Deleted {d["Key"]}')
     if 'Errors' in result:
         for e in result['Errors']:
-            tty.debug('Failed to delete {0} ({1})'.format(
-                e['Key'], e['Message']))
+            ramble.util.logger.logger.debug('Failed to delete {e["Key"]} ({e["Message"]})')
```

Comment:
Missing 'f'. Should be `ramble.util.logger.logger.debug(f'Failed to delete {e["Key"]} ({e["Message"]})')`

Reply:
Good catch. Thanks